backend/src/test/java/com/github/redstylzz/backend/service/PaymentServiceTest.java

```diff
@@ -46,32 +54,38 @@ void shouldThrowExceptionIfCategoryDoesNotExist_OnAdd(){
 
     @Test
     void shouldReturnListAfterAddingPayment_OnAdd() {
-        Payment payment = TestDataProvider.testPayment();
-        String userID = payment.getUserID();
+        String randomUUID = UUID_STRING;
```

Comment:
if you have assigned this value to a static final attribute in your test class you do not need to assign it to another variable to be able to use, this is redundant code polluting your test method